client/src/components/account/admin.tsx

```diff
@@ -1,25 +1,36 @@
+import { getAuth } from "firebase/auth";
+
 import { useEffect, useState } from "react";
 
 import { getRoomsWhereUserISAdmin } from "../../lib/firestore";
+import { app } from "../../lib/firestore/init";
 import { getStreamKey, resetRoom } from "../../lib/server-api";
 
+const auth = getAuth(app);
 /**
  * AdminView renders RoomAdminUI for each room that a user id admin for
  */
 interface AdminViewProps {
   uid: string;
 }
 const Admin: React.FC<AdminViewProps> = ({ uid }) => {
-  const [rooms, setRooms] = useState<undefined | RoomInfo[]>(undefined);
+  const [state, setState] = useState<{
+    rooms?: RoomInfo[];
+    userToken?: string;
+  }>({});
+
   useEffect(() => {
     async function getRooms() {
-      let rooms = await getRoomsWhereUserISAdmin(uid);
-      setRooms(rooms);
+      let [rooms, userToken] = await Promise.all([
+        getRoomsWhereUserISAdmin(uid),
+        auth.currentUser?.getIdToken(),
```

Comment:
I tried to use the user from UserStore at first, but for some reason it couldn't find the `getIdToken` method.